ikasaneip/builder/jar/src/test/java/org/ikasan/builder/FlowBuilderTest.java

```diff
@@ -1530,4 +1530,97 @@ public void test_successful_withMonitor_implied_build()
                 .withDescription("flowDescription")
                 .withMonitor( builderFactory.getMonitorBuilder().withFlowStateChangeMonitor() );
     }
+
+    /**
+     * Test successful flow creation.
+     */
+    @Test
+    public void test_successful_consumer_implied_and_explicit_builds()
+    {
+        BuilderFactory builderFactory = ikasanApplication.getBuilderFactory();
+
+        // implied
```

Comment:
pedantic point; implicit might be the better word? Directly the opposite of explicit